### PR TITLE
 [Reproduce tool] Run Xvfb and blackbox before launching test binaries.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL when you have time. Once this lands test cases with gestures should be working correctly, which was the last major blocker on Linux.

### CODE REVIEW

File: src/local/butler/reproduce.py [hunk 1]
```diff
@@ -132,6 +134,24 @@ def _download_testcase(testcase_id, testcase, configuration):
   return testcase_path
 
 
+def _setup_x():
+  """Start Xvfb and blackbox before running the test application."""
+  environment.set_value('DISPLAY', ':99')
```
Comment (inferno-chromium) [inline src/local/butler/reproduce.py]: Can you move "':99'" on a constant.

Comment (inferno-chromium) [inline src/local/butler/reproduce.py]: It should check platform and bail out if not linux.

File: src/local/butler/reproduce.py [hunk 2]
```diff
@@ -132,6 +134,24 @@ def _download_testcase(testcase_id, testcase, configuration):
...
+
+  xvfb_runner = new_process.ProcessRunner('/usr/bin/Xvfb')
+  xvfb_process = xvfb_runner.run(additional_args=[
+      ':99', '-screen', '0', '1280x1024x24', '-ac', '-nolisten', 'tcp'
+  ])
+  time.sleep(5)  # Allow some time for Xvfb to start.
+
+  blackbox_runner = new_process.ProcessRunner('/usr/bin/blackbox')
```
Comment (inferno-chromium) [inline src/local/butler/reproduce.py]: Maybe output that starting Blackbox and same for starting Xvfb

File: src/local/butler/reproduce.py (hunk 1 above)
Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: Done

Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: Done

File: src/local/butler/reproduce.py (hunk 2 above)
Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: Done

**Pull request merged**